### DISCUSSION TITLE
OpenRadioss workflow

### DISCUSSION THREAD

**Discussion by mishal0488** [OpenRadioss/OpenRadioss#3024] [Q&A]
Hi guys

I am really keen to start learning OpenRadioss. I have been reading up a lot about it when I have had time.
I am keen to understand what is everyone's workflow when developing models for OpenRadioss. 

What comes to my mind first is:
- Create geometry on your CAD package of choice
- Using GMSH the model can be converted into a .rad file with all the data for the geometry

Thereafter what are you doing to provide the additional information for the simulations?

**Comment by SyazwanSamad** [comment]
1. Setup the model using Hyperworks (in native RADIOSS format) or Ansys/LS-Prepost (in LS Dyna format)
2. If LS Dyna format is used 
- use *INCLUDE_RADIOSS to include keywords that aren't available in LS Dyna.
4. Run the solver. If there's no Hyperworks license, I'll convert the Anim files to D3PLOT
5. Use Hyperworks to view the anim file OR use LS-Prepost to view the d3plot files.

**Reply by mishal0488** [reply]
I was looking for a fully open source workflow

**Reply by QiangF** [reply]
A workflow using python https://github.com/nawijn/operato

**Comment by SyazwanSamad** [comment]
I don't think you've ever mentioned about fully open source workflow. But anyway, i do have simulations that are performed in completely open source approach. 
I have my own python scripts to generate my models and defined all the OpenRadioss keywords which is needed for the simulation. However, my python script is not a general preprocessing software, it is intended a very specific simulation.
For postprocessing, I used pyvista. I also used Vortex-radioss to read the T01 files and plot it with matplotlib.
Hope that'll helps

**Comment by MarianBulla** [comment]
Hello @mishal0488,

thanks for your interest in OpenRadioss.

Did you visit the documentation page, here: https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/21397510/Pre+and+Post+Processing+for+OpenRadioss

There are the workflows in commercial software as well as in open source.
You can also check out FreeCAD and other tools, available. 

Hope it helps you ?

Thanks and best regards,
Marian
